### ISSUE TITLE
User-defined config

### ISSUE THREAD

**Issue by automaton8** [ClimbsRocks/auto_ml#62]
Suggest to use a user-defined configuration file for various settings eg base_estimators, search parameters, scoring
and parse with for example https://docs.python.org/2/library/configparser.html

**Comment by ClimbsRocks** [comment]
yeah, that's definitely a good idea. i've generally been trying to design auto_ml to support this, but haven't put in the time to make that a reality yet. 

this'll happen at some point, but prioritized behind some other important features, like implementing support for deep learning, or optimizing how we do hyperparameter optimization.
